### ISSUE TITLE
Standardize presentation of results on tag pages (Research Notes)

### ISSUE THREAD

**Issue by TildaDares** [publiclab/plots2#9540]
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file `app/views/tag/show/_tab_content.html.erb` in the `plots2` repository (press the little pen Icon) and edit the line as shown below. 

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
@@ -7,8 +7,8 @@
   <% if @node_type == "note" %>
     <div class="tab-pane active" id="notes">
       <% if @notes.nil? || @notes.length == 0 %>
-        <p><%= raw translation('tag.show.no_results_found', tag: params[:id]) %>:</p>
-        <p><a href="/search/<%= params[:id] %>">Search for "<%= params[:id] %>"</a></p>
+        <p>Hi! No Research Notes have this tag 🤷, try checking the <b>Questions</b> or <b>Wiki</b> tabs on this page.</p>
+        <p>Or try searching: <a href="/search/<%= params[:id] %>"><%= params[:id] %></a></p>
       <% else %>
         <%= render partial: "notes/notes", locals: { tagname: params[:id] } %>
       <% end %>
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

**Comment by TildaDares** [comment]
@cesswairimu I'm breaking up the tasks in #9510. Are these changes okay?

**Comment by xdshivani** [comment]
I would like to work on it.

**Comment by TildaDares** [comment]
@xdshivani I'm waiting on a reviewer to approve this and when they do you can work on it.

**Comment by TildaDares** [comment]
@RuthNjeri Can you approve this?

**Comment by RuthNjeri** [comment]
Thanks for opening this up @TildaDares,
For the texts, we encourage using the translation method and adding the text in the en.yaml file... have a look at this example issue.. https://github.com/publiclab/plots2/issues/9249

I also have a question about what active params id should be listed in the text, I will ask that in the original issue

**Comment by RuthNjeri** [comment]
Once you update it to used the translation method and have the text within the en.yml, the issue should be good to go 🚀

**Comment by TildaDares** [comment]
@RuthNjeri I've made the changes. I'm still going to have to create another issue to replace the files that use `tag.show.no_results_found`

**Comment by RuthNjeri** [comment]
@TildaDares can that replacement also be done by this issue? I am concerned that if there are files using the old `tag.show.no_results_found` and it is removed, we might not display the correct content for the users...

Alternatively, we could keep the previous ` no_results_found` and have the notes one as ` notes_no_results_found` then after all the changes have been made, we can update the en.yml file with the nesting you have specified for the notes, wiki, or even questions in the future...

What do you think?

**Comment by TildaDares** [comment]
@RuthNjeri I think the second option is better. This way we don't have to change code that uses no_results_found.

**Comment by TildaDares** [comment]
@RuthNjeri I've made the changes.

**Comment by RuthNjeri** [comment]
Thanks, @TildaDares, I think anyone interested can pick it up...


For the emoji on the text, is there a text/symbol for that emoji? Since it will be on a text file and rendered by the HTML tag

**Comment by TildaDares** [comment]
@RuthNjeri I'll change that now.

**Comment by TildaDares** [comment]
@xdshivani Do you still want to work on this?

**Comment by cesswairimu** [comment]
Thanks a lot  @TildaDares :tada:  for opening this

**Comment by TildaDares** [comment]
@DR-FREKE You can work on this.

**Comment by DR-FREKE** [comment]
@TildaDares  okay

**Comment by DR-FREKE** [comment]
@TildaDares @cesswairimu just made a pull request on this

**Comment by cesswairimu** [comment]
Hi @DR-FREKE , I see you already made a first-timer contribution https://github.com/publiclab/plots2/pull/9550, do you mind leaving this to a first-timer?

**Comment by jywarren** [comment]
This looks awesome folks, thanks so much to @TildaDares and to everyone who's supporting!

**Comment by Guneetsinghtuli** [comment]
Hi @cesswairimu @jywarren I would like to work on this issue #9540 , Can you assign this to me ?

**Comment by TildaDares** [comment]
@Guneetsinghtuli Go ahead🎉

**Comment by Guneetsinghtuli** [comment]
@TildaDares @cesswairimu I have made a pull request please have a look and let me know if there are any changes
